packages/databricks-vscode/resources/python/notebook.workflow-wrapper.py

```diff
@@ -0,0 +1,12 @@
+#This file is autogenerated by the Databricks Extension for VS Code
```

Comment:
I know it's annoying but wouldn't we also have to do this for Scala and R?

Reply:
This requires more investigation. I am not sure how module resolution works for scala and what the user expectations are. We can take this as a followup.